h/heaptrack/stone.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# SPDX-FileCopyrightText: © 2025- AerynOS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : heaptrack
+version     : v1.5.0
```

Comment:
Please remove the `v` or update your local boulder to the newest version, which will throw an error if you try to build this.

Reply:
fixed